frontend/app/src/people/widgetViews/parts/StatusPill.tsx

```diff
@@ -55,7 +55,7 @@ const W = styled.div`
   display: flex;
   align-items: center;
 `;
-export default function GithubStatusPill(props: GithubStatusPillProps) {
+export default function StatusPill(props: StatusPillProps) {
   const { status, assignee, style } = props;
```

Comment:
We can delete the status prop since that is no longer being used

Reply:
The application is using assignee prop form that